unumpy/_multimethods.py

```diff
@@ -1770,3 +1870,193 @@ def roll(a, shift, axis=None):
 @all_of_type(ndarray)
 def rot90(m, k=1, axes=(0, 1)):
     return (m,)
+
+
+def _unwrap_default(p, discont=3.141592653589793, axis=-1):
+    nd = ndim(p)
+
+    dd = diff(p, axis=axis)
+
+    slice0 = [slice(None)] * nd
+    slice0[axis] = slice(0, 1)
+    slice0 = tuple(slice0)
+
+    slice1 = [slice(None, None)] * nd
+    slice1[axis] = slice(1, None)
+    slice1 = tuple(slice1)
+
+    ddmod = mod(dd + pi, 2 * pi) - pi
+    ddmod = where((ddmod == -pi) & (dd > 0), pi, ddmod)
+
+    ph_correct = ddmod - dd
+    ph_correct = where(absolute(dd) < discont, 0, ph_correct)
+
+    up_slice0 = p[slice0]
+    up_slice1 = p[slice1] + cumsum(ph_correct, axis=axis)
+
+    up = concatenate([up_slice0, up_slice1], axis=axis)
+
+    return up
+
+
+@create_numpy(_self_argreplacer, default=_unwrap_default)
+@all_of_type(ndarray)
+def unwrap(p, discont=3.141592653589793, axis=-1):
+    return (p,)
+
+
+@create_numpy(_self_out_argreplacer)
+@all_of_type(ndarray)
+def around(a, decimals=0, out=None):
+    return (a, mark_non_coercible(out))
+
+
+round_ = around
+
+
+def _fix_default(x, out=None):
```

Comment:
I think this is the same as [`trunc`](https://numpy.org/doc/stable/reference/generated/numpy.trunc.html).

>The truncated value of the scalar x is the nearest integer i which is closer to zero than x is.

Reply:
It seems to be the same, yes. Refactor the default in terms of `trunc` then?  Something like:

```python
def _fix_default(x, out=None):
    x = trunc(x)

    if out is None:
        return x
    else:
        copyto(out, x)
```

Reply:
Why not just `return trunc(x, out=out)`?

Reply:
You are right! Forgot about the `out` kwarg in ufuncs. 😅